coral/coral/pkg/graphs/resource_models/License.json

```diff
@@ -533,49 +658,457 @@
+                    "label": {
+                        "en": "Notes"
+                    },
+                    "node_id": "0e624c58-6121-11ee-9fd1-0242ac120003",
+                    "sortorder": 4,
+                    "visible": true,
+                    "widget_id": "10000000-0000-0000-0000-000000000005"
+                },
+                {
+                    "card_id": "2bbcab14-60b1-4fd1-9a52-7f9f5690436f",
+                    "config": {
+                        "defaultValue": {
+                            "en": {
+                                "direction": "ltr",
+                                "value": ""
+                            }
+                        },
+                        "i18n_properties": [
+                            "placeholder"
+                        ],
+                        "label": "Letter Data Raw",
+                        "maxLength": null,
+                        "placeholder": {
+                            "en": ""
+                        },
+                        "uneditable": false,
+                        "width": "100%"
+                    },
+                    "id": "e4f97565-0f20-429b-89b8-b2d71ec2e2a1",
+                    "label": {
+                        "en": "Letter Data Raw"
+                    },
+                    "node_id": "a99a4236-68e0-11ee-81c3-0242ac130004",
+                    "sortorder": 1,
+                    "visible": false,
+                    "widget_id": "10000000-0000-0000-0000-000000000001"
+                },
+                {
+                    "card_id": "aa26f6ab-b0fd-48a2-b1a0-ad290f1cd60c",
+                    "config": {
+                        "dateFormat": "YYYY-MM-DD",
+                        "defaultValue": "",
+                        "label": "License Valid To Date",
+                        "maxDate": false,
+                        "minDate": false,
+                        "placeholder": "Enter date",
+                        "viewMode": "days"
+                    },
+                    "id": "823a57a8-f197-45cd-8f60-de9ee5cbcd49",
+                    "label": {
+                        "en": "License Valid To Date"
+                    },
+                    "node_id": "f8765744-69aa-11ee-942a-0242ac130002",
+                    "sortorder": 3,
+                    "visible": true,
+                    "widget_id": "10000000-0000-0000-0000-000000000004"
+                },
+                {
+                    "card_id": "aa26f6ab-b0fd-48a2-b1a0-ad290f1cd60c",
+                    "config": {
+                        "dateFormat": "YYYY-MM-DD",
+                        "defaultValue": "",
+                        "label": "License Valid From Date",
+                        "maxDate": false,
+                        "minDate": false,
+                        "placeholder": "Enter date",
+                        "viewMode": "days"
+                    },
```

Comment:
Here is the Licence Valid from. Its added it here and removed it later just because of order (replying to comment below)